### PR TITLE
Call to action inside nav

### LINKED ISSUE #156: Update Nav element
## Description
Entire nav isn't in a nav element. Not everything has to be, but it seems odd to have an adjacent nav element (the survey) not be included. This will have originally been because in desktop view for the haven, this content is a non-linked phone number.

## Plan
- [x] Move all Nav items into the `Nav>` even if it's a non-linking phone

## Acceptance Criteria
- [ ] All nav items are inside the `nav` tag

### PR DESCRIPTION
**Pull request by katjam**
Fixes #156 

## Description

- Check for cross browser
- Moves cta item inside nav tags

@neontribe/contemplating-action

### CODE REVIEW

**Comment by Nikomus** [comment]
D'oh!
image

I think `justify-content: space-around` is the best bet here.

File: src/styles/layout.scss [hunk 1]
```diff
@@ -349,7 +349,7 @@ nav,
 .nav-item {
   display: flex;
   flex: 1 0 auto;
-  justify-content: center;
+  justify-content: space-around;
```
Comment (Nikomus) [inline src/styles/layout.scss]: I'm not sure this is doing anything here.

**Review by Nikomus — DISMISSED** [review]
Nice layout! 🛰

File: src/styles/layout.scss (hunk 1 above)
Comment (katjam) [reply src/styles/layout.scss]: Yeah - maybe not. I thought that too.

Comment (katjam) [reply src/styles/layout.scss]: Do you think I should delete? I was too :fearful:

Comment (Nikomus) [reply src/styles/layout.scss]: If you want to! It seems there's only one flex child so the spacing isn't going to change. Perhaps the icon and text used to be the direct children of the nav-item.

**Review by Nikomus — APPROVED** [review]
Still nice layout! 🛰

**Pull request merged**